docs/source/commands/supergraphs.mdx

```diff
@@ -57,41 +57,9 @@ You can also pass config via stdin:
 cat ./supergraph.yaml | rover supergraph compose --config -
 ```
 
-### From a Studio variant
-
-You can optionally pass a variant's graph ref to download each subgraph's SDL and compose the supergraph SDL like so:
-
-```bash
-rover supergraph compose --graph-ref platform@staging
-```
-
-You can optionally pass a [YAML configuration file](#yaml-configuration-file) to override specific subgraphs or add a new one.
-This is useful for testing new subgraph schemas before publishing them.
-
-For example, given a `supergraph_override.yaml` file like this:
-
-```yaml title="supergraph_override.yaml" showLineNumbers=false
-subgraphs:
-  products:
-    routing_url: http://localhost:4000
-    schema:
-      file: ./products.graphql
-```
-
-You can override a variant's published `products` subgraph like so:
-
-```bash showLineNumbers=false
-rover supergraph compose \
-  --graph-ref docs-example-graph@current \
-  --config path/to/supergraph_override.yaml
-
-```
-
-Note that you only need to set `routing_url` if you want to change it from the routing URL registered for the subgraph in GraphOS.
-
 ### YAML configuration file
 
-The supergraph configuration file (often referred to as `supergraph.yaml`) includes configuration options for each of your [subgraphs](/federation/building-supergraphs/subgraphs-overview/). The following example file configures a supergraph with two subgraphs (`films` and `people`):
+The supergraph configuration file (often referred to as `supergraph.yaml`) includes configuration options for each of your subgraphs. The following example file configures a supergraph with two subgraphs (`films` and `people`):
```

Comment:
Better to use hover text (which includes a learn more link) rather than peppering links everywhere.